src/PROPOSAL/PROPOSAL/geometry/Box.h

```diff
@@ -30,7 +30,7 @@
 #pragma once
 
 #include "PROPOSAL/geometry/Geometry.h"
-#include <nlohmann/json.hpp>
+#include <nlohmann/json_fwd.hpp>
```

Comment:
`json_fwd.hpp` is directly included here, but not in Cylinder or Sphere,
not even in Geometry.h
one could make this more consistent